### ISSUE TITLE
Cannot get debuginfo on Ubuntu 14.04 with scylla-gdb

### ISSUE THREAD

**Issue by syuu1228** [scylladb/scylla#2360]
*Installation details*
Scylla version (or git commit hash): 1.7
Cluster size:
OS (RHEL/CentOS/Ubuntu/AWS AMI): Ubuntu 14.04

Even scylla-server-dbg installed, we cannot get debugging symbol on scylla-gdb:
```
syuu@takuya-ubuntu14-1:~$ sudo /opt/scylladb/bin/gdb /usr/bin/scylla /var/lib/scylla/coredump/core.scylla-2017-05-08-06\:43\:19-9547
GNU gdb (Ubuntu 7.11-0ubuntu1) 7.11
Copyright (C) 2016 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html>
This is free software: you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.  Type "show copying"
and "show warranty" for details.
This GDB was configured as "x86_64-linux-gnu".
Type "show configuration" for configuration details.
For bug reporting instructions, please see:
<http://www.gnu.org/software/gdb/bugs/>.
Find the GDB manual and other documentation resources online at:
<http://www.gnu.org/software/gdb/documentation/>.
For help, type "help".
Type "apropos word" to search for commands related to "word"...
Reading symbols from /usr/bin/scylla...(no debugging symbols found)...done.
[New LWP 9547]
[New LWP 9553]
[New LWP 9552]
[New LWP 9549]
[New LWP 9548]
[New LWP 9551]
[New LWP 9550]
[New LWP 9555]
[New LWP 9556]
[New LWP 9554]
[New LWP 9557]
[New LWP 9558]
[Thread debugging using libthread_db enabled]
Using host libthread_db library "/lib/x86_64-linux-gnu/libthread_db.so.1".
Core was generated by `/usr/bin/scylla --log-to-syslog 1 --log-to-stdout 0 --default-log-level info --'.
Program terminated with signal SIGABRT, Aborted.
#0  0x00007f2e13f0ee27 in epoll_pwait () from /lib/x86_64-linux-gnu/libc.so.6
[Current thread is 1 (Thread 0x7f2e18399440 (LWP 9547))]
(gdb) bt
#0  0x00007f2e13f0ee27 in epoll_pwait () from /lib/x86_64-linux-gnu/libc.so.6
#1  0x00000000004e1deb in ?? ()
#2  0x00000000004e1f61 in ?? ()
#3  0x000000000051db33 in ?? ()
#4  0x000000000059e87a in ?? ()
#5  0x0000000000459989 in ?? ()
#6  0x00007f2e13e32f45 in __libc_start_main () from /lib/x86_64-linux-gnu/libc.so.6
#7  0x00000000004c99c2 in ?? ()
(gdb) quit
```

It's probably because separate-debug-dir is NOT /usr/lib/debug, we should pass correct PATH for ./configure when building the package:
```
syuu@monster:~/scylla.30$ /opt/scylladb/bin/gdb --configuration
This GDB was configured as follows:
   configure --host=x86_64-linux-gnu --target=x86_64-linux-gnu
             --with-auto-load-dir=$debugdir:$datadir/auto-load
             --with-auto-load-safe-path=$debugdir:$datadir/auto-load
             --with-expat
             --with-gdb-datadir=/opt/scylladb/share/gdb (relocatable)
             --with-jit-reader-dir=/opt/scylladb/lib/gdb (relocatable)
             --without-libunwind-ia64
             --with-lzma
             --with-python=/usr
             --without-guile
             --with-separate-debug-dir=/opt/scylladb/lib/debug (relocatable)
             --with-system-gdbinit=/opt/scylladb/etc/gdb/gdbinit (relocatable)
             --with-babeltrace

("Relocatable" means the directory can be moved with the GDB installation
tree, and GDB will still find it.)
```

**Comment by slivne** [comment]
@syuu1228 - still relevant - or can we close ?

**Comment by slivne** [comment]
@syuu1228 ping

**Comment by slivne** [comment]
@syuu1228 ping ?

**Comment by slivne** [comment]
@syuu1228 - ping ?

**Comment by syuu1228** [comment]
This was scylla-gdb package bug, fixed at newly released scylla-gdb 7.11-2.
